generated/sqlc/db_test.go

```diff
@@ -0,0 +1,154 @@
+package sqlc
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/ory/dockertest"
+	"github.com/ory/dockertest/docker"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+var q *Queries
+
+func TestMain(m *testing.M) {
+	// uses a sensible default on windows (tcp/http) and linux/osx (socket)
+	pool, err := dockertest.NewPool("")
+	if err != nil {
+		logrus.Fatalf("Could not connect to docker: %s", err)
+	}
+
+	cdir, err := os.Getwd()
+	if err != nil {
+		logrus.Fatalf("Could not find current directory: %s", err)
+	}
+
+	fmt.Printf("%v/../../db/schema.sql:/docker-entrypoint-initdb.d/schema.sql", cdir)
+
+	// pulls an image, creates a container based on it and runs it
+	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
+		Repository: "postgres",
+		Tag:        "13-alpine",
+		Env: []string{
+			"POSTGRES_USER=test",
+			"POSTGRES_PASSWORD=test",
+			"listen_addresses = '*'",
+		},
+		Mounts: []string{
+			fmt.Sprintf("%v/../../db/schema.sql:/docker-entrypoint-initdb.d/schema.sql", cdir),
```

Comment:
I think you can compose the first path using the `path.Join`.

Reply:
I prefer to leave this now and handle better later wiith docker comopse file